src/user/user.spec.js

```diff
@@ -55,6 +55,27 @@ describe("user", () => {
     });
   });
 
+  describe(".getAll() :", () => {
+    it("gets all the users created", async () => {
+      const userId1 = await xo.createUser({
+        email: "wayne4@vates.fr",
+        password: "batman",
+        permission: "user",
+      });
+      const userId2 = await xo.createUser({
+        email: "wayne5@vates.fr",
+        password: "batman",
+        permission: "user",
+      });
+      const users = await xo.call("user.getAll");
+      expect(Array.isArray(users)).toBe(true);
+      expect([
+        omit(find(users, { id: userId1 }), "id"),
+        omit(find(users, { id: userId2 }), "id"),
```

Comment:
Why omitting the `id`s?

Reply:
We omit the `id` for the snapshots.

Reply:
Ah yes, I forgot, thanks for the reminder :slightly_smiling_face:

Reply:
Why not just omitting the `ids` from `users` and simply test the  `users` with their snapshot ?